src/python/strelka/scanners/scan_ocr.py

```diff
@@ -22,8 +22,9 @@ def scan(self, data, file, options, expire_at):
         pdf_to_png = options.get('pdf_to_png', False)
 
         if pdf_to_png and 'application/pdf' in file.flavors.get('mime', []):
+            # TODO: Use fitz builtin OCR support which also wraps tesseract
```

Comment:
Not strictly necessary but might be better? A quick glance says this may not be worth it, but we can look into it further later